sdk/core/core-http/lib/coreHttp.ts

```diff
@@ -51,3 +52,4 @@ export { ApiKeyCredentials, ApiKeyCredentialOptions } from "./credentials/apiKey
 export { ServiceClientCredentials } from "./credentials/serviceClientCredentials";
 export { TopicCredentials } from "./credentials/topicCredentials";
 export { Authenticator } from "./credentials/credentials";
+export * from "@azure/core-tracing";
```

Comment:
Is the reason to export this so that the singleton Tracer that is used in core-http be the same as that in the client library and in the customer code?

Reply:
Exactly

Reply:
Exactly

Reply:
What happens when the user uses multiple Azure libraries at the same time. Does using the singleton exported from one library work with the other as long as both libraries use the same version of `core-http`?

Reply:
This should be the case, yes. Part of why we had to do this was because when using rush, the tracing library was imported multiple times (hypothesis is that the symlinks pnpm/rush use is confusing node's module resolution), which made the integration untestable. 

In the real-world, as long as the same core-tracing library is used in every package (which npm should handle when it flattens dependencies) the singleton should work as expected.